### DISCUSSION TITLE
Add a brief docs

### DISCUSSION THREAD

**Discussion by ramanath-codeclouds** [Jamie-Yang/vue3-boilerplate#23] [Ideas]
Please add a documentation on the customizations you made for this project. It'd help us understand this better.

**Comment by KicKerBNU** [comment]
Hello @Jamie-Yang, first of all congrats for the initiative, nice job with vue3-boilerplate.

I agree with @ramanath-codeclouds maybe you can give more info about the idea behind of the vue3-boilerplate, like the benefits of using this strategy instead of vite for example or your point of view of this structure and dependencies. 

Have a good day 🍺
